docs/_posts/2016-01-12-developers-manual.md

```diff
@@ -167,192 +167,6 @@ API access to any registered Issue trackers, e.g. JIRA, RT, Redmine, Mantis, is
 
 Issue tracker managers allow integration with external issue trackers, removing the need to context switch between MISO and said tracker by a user. An example of this feature is in the Project page, where one or more issue IDs can be supplied which will import the issue details from the tracker's API. Currently the only default supported implementation is JIRA, as provided by the [JiraIssueManager](https://github.com/TGAC/miso-lims/blob/develop/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/service/integration/jira/JiraIssueManager.java) class.
 
-### Printers
```

Comment:
This is about writing a manger, which isn't necessary anymore. This documentation doesn't discuss any of the models and most of what's in here is out of date anyway. I've filed a ticket: GLT-1436